Reset the position of controller to the top left

lakpro opened the pull request:
The positioning of the controller was getting overridden by the element.style.
Added "!important" to keep the controller in the top left by default.

lakpro commented:
Hey @igrigorik  @oggy  @paulirish  @jbrains ,

I fixed the following issues : #850 and #851 

Kindly merge this.

PS: (tagging everyone) It's my first PR, not sure what else to do !!

pepijnmm commented:
![image](https://user-images.githubusercontent.com/3217184/131520709-67aeed7d-e0d7-4147-ad6a-275744640bc0.png)
but the top and left are supposed to be overwritten.
the function to override is in tc.videoController.prototype.initializeControls https://github.com/igrigorik/videospeed/blob/8f3d6b60491d44ecbca4c5ffc9efc1af0a50bd39/inject.js#L272

lakpro commented:
Hey @igrigorik @oggy @paulirish @jbrains,

My previous code had a bug about dragging. 
I have removed the " !important " tag in my previous code as it did not let the user drag the controller.

There was some issue in inject.js where it was not coming to the corner due to max function, which I changed to min. I have tested this code on a few websites, and it's working fine.

As pointed out by @pepijnmm, the overridden values also get fixed, which were earlier wrong.

Now controller comes to the top left corner, and it is draggable.

I kindly request you to look into this issue asap as many people are facing difficulty and it's annoying everyone.
